README.md

```diff
@@ -111,6 +114,21 @@ feat(ui): Add `Button` component
           wip: true
 ```
 
+### Required Permissions for [WIP] feature
+
+If you want to use the `[WIP]` feature, you need to grant the
+`pull-requests: write` permission to the GitHub Action. This is because the
+action will update the status of the PR.
+
+```yml
+name: "Lint PR"
+
+# ...
+
+permissions:
+  pull-requests: write
+```
+
```

Comment:
Looks pretty good!

Maybe we can move the whole config option for `wip` to this block, similar to what we've done with the legacy configuration below? I'm just worried that users stop reading at the `wip` option above and don't notice that permissions need to be changed (even though it's the very next headline).

By doing this, we can make sure we're educating users about the required permissions as we're introducing the feature. Maybe you can reuse the description from above here and adjust the headline accordingly to match the whole section?

I just noticed the "Note that a second check will be reported if this is enabled." part is no longer valid, can you remove this as part of moving the text?

Thank you so much! 🙏

Reply:
Hey @amannn,

please review again and take a look at:

- [`1ebd749` (#215)](https://github.com/amannn/action-semantic-pull-request/pull/215/commits/1ebd749c2a2e3c7413f0d3e19df1d9d4e6790fa6)
[`f030897` (#215)](https://github.com/amannn/action-semantic-pull-request/pull/215/commits/f030897c39c6656af87f5200b0739d52d7582f75)

Reply:
Looks great, thanks! I've included a commit with a few further README improvements that I had on my list to include them in the next release.